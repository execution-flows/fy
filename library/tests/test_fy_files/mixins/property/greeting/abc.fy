import datetime

property greeting: datetime
