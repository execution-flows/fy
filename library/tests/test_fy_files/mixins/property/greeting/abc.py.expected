import datetime

import abc


class With_Greeting_PropertyMixin_ABC(abc.ABC):
    @property
    @abc.abstractmethod
    def _greeting(self) -> datetime:
        raise NotImplementedError()
